src/arkode/arkode_arkstep_io.c

```diff
@@ -1767,10 +1741,8 @@ int ARKStepSetPredictorMethod(void* arkode_mem, int pred_method)
      has been supplied */
   if ((pred_method == 5) && (step_mem->stage_predict != NULL))
   {
-    arkProcessError(ark_mem, ARK_ILL_INPUT, "ARKODE::ARKStep",
-                    "ARKStepSetPredictorMethod",
-                    "predictor 5 cannot be combined with user-supplied stage "
-                    "predictor");
+    arkProcessError(ark_mem, ARK_ILL_INPUT, __LINE__, __func__,
+                    __FILE__, "predictor 5 cannot be combined with user-supplied stage predictor");
```

Comment:
Yeah, I was staring at that earlier, since I was certain that we'd removed it in an earlier PR. But after looking through develop, etc., it was still everywhere, so I convinced myself that we deprecated it but had deferred removal. 

Maybe we'd removed it, but it got put back in when merging some other PR?

I'm all for removal, if others were also thinking it had been (or should have been) removed.

Reply:
I think we might have deferred removal while other PRs were getting sorted out. Looking back at the change log, we deprecated this option and the "bootstrap" predictor in v6.0.0 (see text below) and should remove them for this release. 

> Deprecated ARKODE nonlinear solver predictors: specification of the ARKStep "bootstrap" or "minimum correction" predictors (options 4 and 5 from ARKStepSetPredictorMethod), or MRIStep "bootstrap" predictor (option 4 from MRIStepSetPredictorMethod), will output a deprecation warning message. These options will be removed in a future release.

Reply:
I am pretty sure I removed this code before but in all of the merges it showed back up.

Reply:
I don't remember a PR removing these predictor options but I think it's ok to have a separate RP to take care of that.

Reply:
The options were removed in the PR that removed deprecated code. See https://github.com/LLNL/sundials/pull/360/files#diff-9a5906ded891b9f50009d5b77b22868d60ee3306e0500d2334c9f1a04535da58L1459. The line this comment is attached to seems to have been missed or made its way back in through the merges.